content/strategy-goals/strategy/code-intelligence/index.md

```diff
@@ -35,19 +35,19 @@ At the moment maturity varies depending on the area of ownership:
+- **Code intelligence platform:** We're currently working on delivering an out-of-the-box precise code intelligence solution to a small number of eager customers. This allows customers to easily set up precise code intelligence for Go, Java, Scala, Kotlin and TypeScript repositories. This also enables automatic indexing of a repository's dependencies, which means users are now be able to navigate and find references across repositories and dependencies alike.
 
-- **Code navigation:** Baseline features are implemented but have a considerable amount of debt and room for improvement. We're currently focusing on researching and implementing features that will make the overall navigation experience faster and more intuitive.
+- **Code navigation:** Baseline features are implemented but have a considerable amount of debt and room for improvement. We're currently focusing on fixing and improving our core workflow features, with the goal of making the code navigation experience feel fast and reliable.
 
 In the last few months we’ve:
 
-- Added precise support for Python and revamped our TypeScript support.
-- Improved our search-based navigation experience by implementing efforts like [Rockskip](https://docs.sourcegraph.com/code_intelligence/explanations/rockskip) and [Squirrel] (https://docs.google.com/document/d/10YHHydQXmPODzeDDSAm_audheahdc8NjCqQPBV_wcN0)
-- Shipped the [executors service](https://docs.sourcegraph.com/admin/executors) that powers [auto-indexing](https://docs.sourcegraph.com/code_intelligence/explanations/auto_indexing) at on-prem customers that use GCP or AWS.
+- Improved our search-based navigation experience by implementing efforts like [Rockskip](https://docs.sourcegraph.com/code_intelligence/explanations/rockskip) and [Squirrel] (https://docs.google.com/document/d/10YHHydQXmPODzeDDSAm_audheahdc8NjCqQPBV_wcN0) for the top used languages.
+- Shipped the [executors service](https://docs.sourcegraph.com/admin/executors) that powers [auto-indexing](https://docs.sourcegraph.com/code_intelligence/explanations/auto_indexing) to unlock precise cross repository and dependency navigation.
+- Launched [SCIP](https://about.sourcegraph.com/blog/announcing-scip), our new code indexing format, to address shortcomings of the LSIF format and allow us to deliver and iterate on indexers faster. This also paves the way for long awaited features like incremental indexing and cross-language navigation.
 
 Recent key learnings:
 
-- In alignment to the company's pivot towards managed instances, establishing a pricing strategy for the executors service will be key part of moving Auto-indexing into Beta.
+- In alignment to the business strategy, making executors and auto-indexing available by default on managed instances and ensuring will be key part of moving Auto-indexing into Beta.
```

Comment:
I think this is missing some why, why is it key?  executors are a tool to get us to where?